launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -396,8 +396,11 @@ QStringList MinecraftInstance::extraArguments()
         agent->library()->getApplicableFiles(runtimeContext(), jar, temp1, temp2, temp3, getLocalLibraryPath());
         list.append("-javaagent:"+jar[0]+(agent->argument().isEmpty() ? "" : "="+agent->argument()));
     }
-    if (version->getModLoaders().value() & ResourceAPI::Quilt && settings()->get("DisableQuiltBeacon").toBool()) {
-        list.append("-Dloader.disable_beacon=true");
+
+    {
+        const auto& loaders = version->getModLoaders();
```

Comment:
```suggestion
        const auto loaders = version->getModLoaders();
```

I'm pretty sure this is just an implicit conversion for no reason

Reply:
Not really. I just have muscle memory when I use a getter function to only take it by reference. But we are dealing with an enum / integers here, so it's really unnecessary.